org.eclipse.debug.ui/ui/org/eclipse/debug/ui/InspectPopupDialog.java

```diff
@@ -171,6 +171,17 @@ public void updateComplete(IViewerUpdate update) {
 		return fTree;
 	}
 
+	/**
+	 * The viewer is initialized after the dialog is constructed, so only after open
+	 * is called.
+	 *
+	 * @since 3.17
+	 * @return the tree model viewer used by the dialog.
+	 */
+	public TreeModelViewer getTreeModelViewer() {
```

Comment:
I see this API warning -
InspectPopupDialog.getTreeModelViewer() has non-API return type 
TreeModelViewer
We cannot return an internal class from an API.

Reply:
We can declare the return type to be org.eclipse.jface.viewers.TreeViewer, can't we?